proposals/forms.py

```diff
@@ -832,7 +832,8 @@ def __init__(self, *args, **kwargs):
         self.request = kwargs.pop("request", None)
 
         # Needed for validation
-        self.final_validation = kwargs.pop("final_validation", None)
+        final_validation = kwargs.pop("final_validation", None)
+        self.final_validation = None
```

Comment:
I think I intended to add a comment along the lines of

"we save this variable for later, but want it set for None for this part of the validation"

Because as it stands this is kind of confusing.